sjb/hack/determine_install_upgrade_version.py

```diff
@@ -3,6 +3,16 @@
 import os.path
 import rpmUtils.miscutils as rpmutils
 
+# Remove duplicate packages from different repositories
+def remove_duplicate_pkgs(available_pkgs):
```

Comment:
unfortunately no. The thing is that the pkgs that point to the same verions are pointing to a different repo, that why `set()` wont treat them as dups.